rasa/shared/importers/rasa.py

```diff
@@ -54,10 +58,18 @@ async def get_stories(
             exclusion_percentage,
         )
 
+    async def get_conversation_tests(self) -> StoryGraph:
+        """Retrieves conversation test stories (see parent class for full docstring)."""
```

Comment:
the alternative is to filter in `get_stories` based on `use_e2e`. Considering [this](https://martinfowler.com/bliki/FlagArgument.html) and also to avoid anything breaking I decided to go for the explict extra method